core/itwin-connector-framework/src/ConnectorRunner.ts

```diff
@@ -87,20 +87,20 @@ export class BridgeRunner {
         const argFile = keyVal[0].substr(1);
         if (!fs.existsSync(argFile))
           throw new Error("Error file {argFile} does not exist.");
-        BridgeRunner.parseArgumentFile(argFile, bridgeJobDef, serverArgs, bankArgs);
+        ConnectorRunner.parseArgumentFile(argFile, connectorJobDef, serverArgs, bankArgs);
         continue;
       }
       const argName = keyVal[0].trim();
       switch (argName) {
-        case "--fwk-input": bridgeJobDef.sourcePath = keyVal[1].trim(); break;
-        case "--fwk-output": bridgeJobDef.outputDir = keyVal[1].trim(); break;
-        case "--fwk-bridge-library": bridgeJobDef.bridgeModule = keyVal[1].trim(); break;
+        case "--fwk-input": connectorJobDef.sourcePath = keyVal[1].trim(); break;
+        case "--fwk-output": connectorJobDef.outputDir = keyVal[1].trim(); break;
+        case "--fwk-connector-library": connectorJobDef.connectorModule = keyVal[1].trim(); break;

```

Comment:
I know this was bridge-library to be consistent with the native connectors, is this still a concern/goal?  @c-macdonald  @jffmarker

Reply:
I am not particularly concerned with the JS connectors having the same exact arguments... I don't think there's any way we can re-use the raw EXE call by just swapping EXE names. However, this brings up a good point that we need to coordinate with the Instant On folks to update how they invoke the JS Connectors. We have plans in Q3 to revise how we invoke JS connectors as well.

Reply:
okay, sounds good.